src/main/flight/pid.c

```diff
@@ -205,21 +209,18 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
     static float previousRateError[2];
     static float previousSetpoint[3];
 
-    if(FLIGHT_MODE(HORIZON_MODE))
-        calcHorizonLevelStrength(pidProfile);
```

Comment:
@ledvinap The horizonStrenght is already like that. One axis with highest deflection (mostDeflected) determines the total strength. What I meant to say is to only perform once on first axis. pidLevel() is being looped twice on ROLL and PITCH. But once you did ROLL you already know your strength